### PR TITLE
To use -scp option for pscp in Download-RemoteFile

### PR DESCRIPTION
**Pull request by prashantmhetar**
Verified with running BVT test:

[LISAv2 Test Results Summary]
Test Run On           : 03/11/2019 08:25:17
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:3

   ID TestCaseName                                                                TestResult TestDuration(in minutes)
---------------------------------------------------------------------------------------------------------------------
    1 BVT-VERIFY-BASH-HISTORY                                                           PASS                 0.68

Logs can be found at F:\LISAv2\MN63\TestResults\2019-11-03-01-25-13-8831

### CODE REVIEW

File: Libraries/TestHelpers.psm1 [hunk 1]
```diff
@@ -163,7 +163,7 @@ function Download-RemoteFile($downloadFrom, $downloadTo, $port, $file, $username
 				$downloadStatusRandomFile=$args[7];
 				Set-Location $curDir;
 				Set-Content -Value "1" -Path $args[6];
-				Write-Output "yes" | .\Tools\pscp -i .\ssh\$sshKey -q -P $port $username@${downloadFrom}:$testFile $downloadTo;
+				Write-Output "yes" | .\Tools\pscp -scp -v -i .\ssh\$sshKey -q -P $port $username@${downloadFrom}:$testFile $downloadTo;
```
Comment (chvalean) [inline Libraries/TestHelpers.psm1]: please remove the verbose flag

**Review by chvalean — CHANGES_REQUESTED** [review]

**Comment by chvalean** [comment]
was this change in the old code to use scp instead?

File: Libraries/TestHelpers.psm1 (hunk 1 above)
Comment (prashantmhetar) [reply Libraries/TestHelpers.psm1]: removed the verbose flag

**Comment by prashantmhetar** [comment]
> was this change in the old code to use scp instead?

yes, using -scp as default for all platforms

**Review by chvalean — APPROVED** [review]

**Pull request merged**